srv/GraspPlanning.srv

```diff
@@ -0,0 +1,27 @@
+# Requests that grasp planning be performed for the target
+# returns a list of grasps to be tested and executed
+
+# the planning group used
+string group_name
+
+# the object to be grasped
+CollisionObject target
+
+# the name that the support surface (e.g. table) has in the collision map
+# can be left empty if no name is available
+string collision_support_surface_name
```

Comment:
while creating this message, should we make this an array? (Or maybe even a CollisionObject array?) This always seemed a bit underspecified to me, and I know I've at least once ran into the issue that an object is sitting half on one support surface and half on another.